lib/velocity-animate-shim.js

```diff
@@ -2,16 +2,17 @@
 // requires window. Note that this just no-ops the components so
 // that they'll render, rather than doing something clever like
 // statically rendering the end state of any provided animations.
-if (typeof window !== 'undefined') {
+if (process.env.NODE_ENV === 'test' ||  typeof window === 'undefined') {
```

Comment:
This chaining of `process` will need to be more defensive of `process` not existing, I believe. Let me know if you want to update this PR or I can take it over.